makefile

```diff
@@ -114,6 +118,19 @@ test-backend-local: backend/api-service/src/api/openapi.yaml
 	cd backend/api-service/src && FIRESTORE_EMULATOR_HOST=localhost:9090 go test -tags=emulator -v
 	docker stop firestore-emulator
 
+FIREBASE_SA=$(shell gcloud --project=$(PROJECT_ID) iam service-accounts list --filter="displayName=firebase-adminsdk" --format="value(email)")
```

Comment:
I'm a little bit confused about this part -- does every GCP project already have a service account named `firebase-adminsdk` by default? Or is this a new prerequisite?

Reply:
It's created when you enable Identity Platform/Firebase

Reply:
Ok cool, thanks for clarifying

Reply:
You can find it at `https://console.firebase.google.com/project/{PROJECT_ID}/settings/serviceaccounts/adminsdk`